OpenRA.Mods.Common/UtilityCommands/PngSheetExportMetadataCommand.cs

```diff
@@ -0,0 +1,39 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.IO;
+using System.Linq;
+using OpenRA.FileFormats;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	public class PngSheetExportMetadataCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--png-sheet-export"; } }
+
+		bool IUtilityCommand.ValidateArguments(string[] args)
+		{
+			return args.Length == 2;
+		}
+
+		[Desc("PNGFILE YAMLFILE", "Export png metadata to yaml")]
+		void IUtilityCommand.Run(Utility utility, string[] args)
+		{
+			using (var s = File.OpenRead(args[1] + ".png"))
```

Comment:
Please keep the extension in the arg, then you can use `System.IO.Path.ChangeExtension` to switch it to yaml.